torch/onnx/utils.py

```diff
@@ -120,15 +120,22 @@ def _split_tensor_list_constants(g, block):
                   .setType(ListType.ofTensors()))
             node.output().replaceAllUsesWith(lc)
 
-def _optimize_graph(graph, operator_export_type, _disable_torch_constant_prop=False, fixed_batch_size=False, params_dict=None):
-    # Inline everyting
+
+def _optimize_graph(graph, operator_export_type, _disable_torch_constant_prop=False, fixed_batch_size=False,
+                    params_dict=None, use_new_jit_passes=False):
+    # Inline everything
     torch._C._jit_pass_inline(graph)
 
     # Remove fork/wait nodes
     torch._C._jit_pass_inline_fork_wait(graph)
     torch._C._jit_pass_lint(graph)
 
-    torch._C._jit_pass_remove_inplace_ops(graph)
+    if not use_new_jit_passes:
+        torch._C._jit_pass_remove_inplace_ops(graph)
+    else:
+        torch._C._jit_pass_lower_all_tuples(graph)
+        torch._C._jit_pass_remove_inplace_ops(graph)
```

Comment:
looks like the new pass `_jit_pass_onnx_remove_inplace_ops_for_onnx` is not being used anywhere